### ISSUE TITLE
Bind-Event-*blank* with identifiers

### ISSUE THREAD

**Issue by TheMallen** [Shopify/twine#56]
I'd be interested in trying to add the ability to use `bind-event-click` and friends with callable identifiers, something like `bind-event-click="myObj.method"` or even `bind-event-click="module.decorator(myObj.method)"`. The advantage here is that you would only have to evaluate the string once, and it would be much more trivial to compose callbacks together. I'm not sure how it would be implemented but if other people like this idea I'd be down to take a look at doing so. Naively, I feel like you could simply check if the result of evaling the expression is a function and if so bind that function to the event directly? That's without looking too hard at how it's actually implemented yet though.

TLDR:
I want to make this possible
```
<input bind-event-keydown="coolFilter(debounce(viewModel.doAThing))">
```

Instead of having to do
```
<input bind-event-keydown="coolFilter(debounce(viewModel.doAThing))(event)">
```
or other such awkward stuff.
